src/module/actor/character/apps/abc-picker/app.svelte

```diff
@@ -26,26 +40,15 @@
         actor.createEmbeddedDocuments("Item", [{ ...item.toObject(), _id: null }]);
         foundryApp.close();
     };
-
-    /** Search list and show or hide according to match result. */
-    const searchItems: KeyboardEventHandler<HTMLInputElement> = (event) => debouncedSearch(event.currentTarget.value);
-    const debouncedSearch = fu.debounce((query: string) => {
-        const regexp = new RegExp(RegExp.escape(query.trim()), "i");
-        for (const row of foundryApp.element.getElementsByTagName("li")) {
-            const name = row.innerText ?? "";
-            const originalName = row.dataset.originalName;
-            row.hidden = !regexp.test(name) && !(originalName && regexp.test(originalName));
-        }
-    }, 200);
 </script>
 
 <search>
     <i class="fa-solid fa-search"></i>
-    <input type="search" spellcheck="false" placeholder={searchPlaceholder} onkeyup={searchItems} />
+    <input type="search" spellcheck="false" placeholder={searchPlaceholder} bind:value={searchQuery} />
```

Comment:
This doesn't respond to keydown events, right?

Reply:
Ah, you mean specifically keydown as opposed to just keyup, rather than compared to blur. Yes, this will do the search as soon as the key is down. Same way the formula picker currently does it. That said before it was using a debounce, so the keyup just served to increase the debounce window, which is more useful when you're doing dom manipulation.